messaging/kafka-strimzi-avro-reactive-messaging/src/test/java/io/quarkus/ts/messaging/strimzi/kafka/reactive/StrimziKafkaAvroGroupIdIT.java

```diff
@@ -5,12 +5,15 @@
 import io.quarkus.test.bootstrap.KafkaService;
 import io.quarkus.test.bootstrap.RestService;
 import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnQuarkusSnapshot;
 import io.quarkus.test.services.KafkaContainer;
 import io.quarkus.test.services.QuarkusApplication;
 import io.quarkus.test.services.containers.model.KafkaVendor;
 
 @Tag("QUARKUS-1089")
 @QuarkusScenario
+// TODO https://github.com/quarkusio/quarkus/issues/25814
+@DisabledOnQuarkusSnapshot(reason = "Apicurio 2.2.3.Final used in upstream is not compatible with Quarkus upstream")
```

Comment:
According to this message, Quarkus upstream uses Apicurio 2.2.3.Final and is incompatible with it in the same time. I assume, that something is wrong here.

Reply:
Comment updated !